src/open_inwoner/accounts/templates/registration/login.html

```diff
@@ -3,7 +3,9 @@
 
 
 {% block header_image %}
+<aside aria-label="{% trans "Header afbeelding" %}">
     <img class="main-image" src="{{ hero_image_login }}" alt="Header image">
```

Comment:
Can add a proper alt text here describing the content of the image?


 > The alt attribute holds a text description of the image, which isn't mandatory but is incredibly useful for accessibility — screen readers read this description out to their users so they know what the image means. Alt text is also displayed on the page if the image can't be loaded for some reason: for example, network errors, content blocking, or linkrot.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img